### DISCUSSION TITLE
[Grant] dReader

### DISCUSSION THREAD

**Discussion by svc01-metaplex** [metaplex-foundation/dao#4] [Community Grant Submissions]
# dReader

## Submission details

Submitter: d-reader-josip
Email: josip.volarevic@dreader.io
Submitter wallet: 7aLBCrbn4jDNSxLLJYRRnKbkqA5cuaeaAzn74xS7eKPD
Beneficiary wallet: 7aLBCrbn4jDNSxLLJYRRnKbkqA5cuaeaAzn74xS7eKPD
$MPLX requested: 1100000

## Team

We're a team of 7, ranging from developers and designers to project managers and creators.
Majority of competition fails because they're either a development team, or creators, or something else entirely.
We are the correct mix of everything 💯

Josip Volarević (Founder & Full stack developer)
- 5+ years of experience in development agencies and as a freelancer
- 2 years in the Solana ecosystem as a developer
- lead a web3 startup as a CTO to its first 1M revenue generated

Adam Jeffcoat (Artist & Illustrator)
- 20+ years in the comic industry
- 2+ years in the Solana ecosystem as a creator
- Emmy Award winning writer & animation director
- portfolio of clients like Disney, Marvel & Amazon Studios
- Net worth of the "Niko and the Sword of Light" franchise +15m

James Roche (Co-founder & Writer)
- 10+ years in the comics industry
- 2+ years in the Solana ecosystem as a creator
- published over a dozen of comics

Karlo Mijoč (Co-founder & Business developer)
- 2+ years in the Solana ecosystem
- 2+ years in product development

Mate Žaja (Creative director & UI/UX designer)
- 10+ years of experience in creative agencies and as a freelancer

Athar Mohammad (Full stack developer)
- 3+ years of experience as a developer
- 2+ years in the Solana ecosystem as a developer
- contributed to core Solana repositories

Luka Crnogorac (Mobile developer)
- 5+ years of experience in development agencies and as a freelancer
- 2+ years of experience as a mobile developer (Flutter)

To bring in more perspective on "Why You?"...
- We're VERY well connected with all parts of the Solana ecosystem: degens, creators, developers, and founders
- We're in direct communication with many entities/projects on Solana: Mad Lads, Tensor, DAA, Heist, Solana Mobile, Solana Foundation, Solana Collective, Superteam, Deanslist etc.
- We're currently working on 7-8 comics for top collections on Solana: Heist, Tensor, DAA, Geckos, and a few more that are still under NDA
- We have been blessed with a Solana Mobile grant in the past and have executed on it successfully
- We have won multiple hackathons: Grizzlython, Sandstorm, Encode etc.
- We know how the ecosystem breathes and moves

## Abstract

dReader is a platform which enables comic readers and collectors to discover, read, collect, and trade digital comics.

We're building it as a team of 7, ranging from developers, designers, marketeers, and all the way to writers and Emmy award winning animators and illustrators.
Purpose of this grant is to get funds which would help us with the adoption of the product through the execution of our go-to-market strategy.

static presentation: https://drive.google.com/file/d/1mzSaD7yob-jtmBaWGtwBlHkaVhuJk_8w/view?usp=sharing
important links: https://dreader.io/links

## Description

We have been working on the project for 14 months. We have welcomed 400 verified users and 60+ comic creators since our private alpha 2 months ago.

During this time, we have come to the following CAC (customer acquisition cost) values:
- onboarding web2 comic creators from Webtoons & Global Comix = $50 CAC
- onboarding web3 comic creators from Ethereum, Tezos, and other chains = $30 CAC
- onboarding web3 comic collectors from ImmutableX (VeVe) = $50 CAC

Onboarding web2 and web3 (non VeVe) readers/collectors is hard to estimate since there are too many factors to consider. Mostly, new users will come as a direct product of attracting specific creators to dReader which bring their audience.

For example, Heist can get us 200 new users, while from an indie Webtoon publisher we can expect 10 new users.

Main takeaways, investing into dReader:
- with $10,000 we can expose 200 web2 creators to Solana
- helps us attract creators and collectors from other chains
- opens jobs for creators on Solana (over 50 paid gigs with artists)
- keeps 7 solid professionals within the ecosystem which create value (we have helped over 8 projects/individuals raise over $70,000 in the last year)

Additional points:
- We're giving away some of our comic NFTs for free which is not cheap in a bull market.
- We expect that our users will mint at least 100,000 NFTs in the next 12 months which is about $200,000 fees paid by our users with the current $SOL price.
- $80,000 out of $200,000 mentioned above will go directly to Metaplex (0.01 $SOL per mint)

- 10% of each comic NFT collection will be minted directly by us and put into a "Comic Vault".
- As a yearly event we'll sell all the comics from the vault as randomized "mystery packs”.
- This means that in a year we'll mint at least 10,000 NFTs and pay about $6,000 to Metaplex (0.01 $SOL per mint)

When everything is considered, we can confidently say that...
- Metaplex 0.01 $SOL fee per mint is killing innovation in our specific use-case
- The money which this grant would provide to us (in case we get it) will go back to Metaplex org through mint fees
- We aim to onboard tens of thousands of users which will mint Solana (Metaplex) NFTs.
- In the long run, if we succeed with our mission, Metaplex will profit from our solution.

On the "Protocol Fees | Metaplex Docs" website Metaplex officially states: "Our goal is for mint fees to be minimally disruptive and promote the growth and usage of the protocol."
The current bull market fees disrupt the adoption of our app significantly. This is especially noticed on "free" mints which cost almost $2.

The problem this grant solves:
AWS & GCP give free credits to startups for ~2 years. Apple & Google take 15% instead of 30% for mobile apps with less than 1M yearly revenue.
In a similar way, a Metaplex grant would help our startup stand on two feet.

side note: cNFTs are not an option (yet!) for multiple reasons: no enforced royalties, metadata updates not handled by indexers, lack of Program support (AH and CM) etc.

## Milestones (optional)

Milestone 1 - 400,000 $MPLX to onboard 400 creators to the platform (ETA: 12 months)
- 300 web2 creators from platforms like Webtoons and Global Comix
- 100 web3 creators from Solana and other chains

Milestone 2 - 500,000 $MPLX to onboard 5,000 verified users onto the platform (ETA: 9 months)

Milestone 3 - 200,000 $MPLX to make success stories with artists on dReader (ETA: 8 months)
- make at least 3 comics generate more than $5,000 in profits
- make at least 3 indie creators generate more than $2,000 in profits
- make at least 3 web2 creators generate more than $2,000 in profits

KPI is straightforward: number of creators, comics, and verified users on the platform, followed by profits and volume generated by specific collections.

ETA 12 months to execute on the grant scope.

**Comment by m-luketin** [comment]
The dReader guys killing it!

**Comment by topicanica** [comment]
Josip is one of the hardest working and one of the most ambitious people I know. Him and his team have been working non-stop to make this project come to life.

**Comment by thearyanag** [comment]
Josip and his team are goated.

**Comment by jrej-dev** [comment]
Very promising project with a hard working team. 
They understand both the Solana and comics space which is a perfect fit for dreader.
The community around the project is growing which shows their consistent efforts.

**Comment by pratikbuilds** [comment]
Josip has been grinding non-stop and has been in the ecosystem for a long time i would love to see them being successfull

**Comment by dannpl** [comment]
Really amazing!

**Comment by cdhiraj40** [comment]
Josip, Athar, and the entire dReader team have been absolutely killing it with their dedication and hard work. I would genuinely love to see their continued success!

**Comment by nborovic** [comment]
Great product and an even better team behind it 🚀

**Comment by joeymeere** [comment]
One of the hardest working teams in the eco.

Help these guys out!

**Comment by creativedrewy** [comment]
This team is one of the best in ecosystem! They are building something unique and have already proved they can make cool things happen.

Helping them succeed is a win for Metaplex, a win for Solana, and a win for web3 in general. Let's get them this grant!

**Comment by ChoutzLAN** [comment]
These lads are talented and dedicated. They deserve support!

**Comment by MetaplexFoundation** [comment]
Thank you for the grant proposal submission! The community feedback period will remain open until December 4, 2023.

**Comment by Barbarian83** [comment]
I would love to see this project go

**Comment by jim4067** [comment]
Would really love to see this project take off in 2024.

**Comment by Aiyualive** [comment]
This team has been grinding through the bear market! One of the hardest working teams out there!

**Comment by Michaelsulistio** [comment]
Josip and the dReader team are one of the hardest working and talented teams in the ecosystem with a product that aims to grow the pie and attract the masses. The dReader mobile app, for example, is an awesome web3 consumer product that stands out with its great UX, a growing user base, and an ambitious vision for the future.

**Comment by noamrubin22** [comment]
I totally support! Would love to see this project take off :)

**Comment by pdahlberg** [comment]
This team deserves a grant for sure! They are executing very well on a an end user product which actually can bring new people to the ecosystem. On top of that, with a great mobile app they are ready to meet readers and fans wherever they are.

**Comment by robbestad** [comment]
Great product, great team! 👍

**Comment by humphreydefirst** [comment]
Best of luck guys, MPLX give them the support 🙏🏾

**Comment by MetaplexFoundation** [comment]
The community feedback period for this grant submission has now ended. The submission has passed the review phase and a vote will be initiated in the Metaplex DAO (dao.metaplex.com) within a few days. Please stay tuned for an announcement for when the voting period begins.
